src/content/troubleshooting/faq/skip-running-builds.mdx

```diff
@@ -0,0 +1,11 @@
+---
+layout: "../../../layouts/FAQLayout.astro"
+sidebar: { hide: true }
+title: Can I skip running builds for the main branch to save snapshots?
+section: "usage"
+sectionOrder: 3
+---
+
+# Can I skip running builds for the `main` branch to save snapshots?
+
+Avoiding Chromatic runs on the `main` branch isn't practical as feature branches depend on it for baseline comparisons. Some users only run builds on the main branch after making changes on feature branches. This ensures the main branch's baseline always reflects the latest merged changes.
```

Comment:
@hannahpx I'm not sure I totally get this:
> Some users only run builds on the main branch after making changes on feature branches. This ensures the main branch's baseline always reflects the latest merged changes.
Is the idea that you don't run Chromatic on feature branch and only run it on main?

Reply:
hey @winkerVSbecks this was written quite messy so apologies! And yes

I think this works better - clearer and simpler:

It is advised to run builds on the `main` branch - avoiding doing this isn't practical as feature branches depend on it for baseline comparisons. 


what do you think?

Reply:
And would you drop this part?

> Some users only run builds on the main branch after making changes on feature branches. This ensures the main branch's baseline always reflects the latest merged changes.